brave-lists/debounce.json

```
@@ -1,6 +1,7 @@
 [
   {
     "include": [
+      "*://cc.mashable.com/*",
```

Let's make this rule specific to the current version of their endpoint:
```suggestion
      "*://cc.mashable.com/v1/*",
```